### ISSUE TITLE
Ubuntu VM crash

### ISSUE THREAD

**Issue by Giancarlo1991** [UnitTestBot/UTBotCpp#687]
**Description**
My VM with Ubuntu v20.04.6 LTS crashes afte rstarting the install script and gets a kernel panic, which i can't recover.
**To Reproduce**
1. VM with Ubuntu v.20.04.6
2. Start install script 2024.3.0

**Expected behavior**
VM crashes
**Actual behavior**
Kernel panic

**Environment**
Windows 11 host, Ubuntu guest virtual box
**Additional context**

**Comment by ChengYen-Tang** [comment]
Hi @Giancarlo1991,
I also have the same problem. Do you resolve this?

**Comment by Giancarlo1991** [comment]
Yeah, i found out that you can't just increase the vm disk size via virtual box. My initial idea was to increase the VM disk space size because it told me I am running out of space but only via virtual box it wasn't enough. You also have to use a partition editor like https://sourceforge.net/projects/gparted/ . Only increasing the size via VM increases only the possible size the VM can have but not the actual. So first via Virtual Box and then with the partition editor.

**Comment by ChengYen-Tang** [comment]
I get this error
![image](https://github.com/user-attachments/assets/cf9b3987-9869-4d46-9d32-5e7b1f562b54)
Then I can't boot my ubuntu
![image](https://github.com/user-attachments/assets/a332d77e-2b90-4bee-8d41-a478a26b200e)

**Comment by Giancarlo1991** [comment]
Yeah, i had this too. It seems the shared libraries were destroyed. I tried fixing it with booting from a new ubuntu version and mounting the old partitions but all shared libs were destroyed and i couldn't use any commands so i threw the vm away...

**Comment by ChengYen-Tang** [comment]
So, you don’t have a solution to this problem?
Or do you mean, don't use VM?

**Comment by Giancarlo1991** [comment]
Sorry, I don't have a solution.

**Comment by momo-tong** [comment]
I also have the same problem.  I see some code changed *so file in utbot_run_system.sh, but don't know how to modify.
Also I use utbotcpp in VM.
Looking forward to the author's reply.
  ```
# Updating libm.so so that it contains valid path to libmvec_nonshared.a
echo "/* GNU ld script */
OUTPUT_FORMAT(elf64-x86-64)
GROUP ( /$X86_LIBS/libm.so.6  AS_NEEDED ( $UTBOT_ALL/debs-install/usr/$X86_LIBS/libmvec_nonshared.a /$X86_LIBS/libmvec.so.1 ) )" > $UTBOT_ALL/debs-install/usr/$X86_LIBS/libm.so

# Updating libc.so so that it contains valid path to libc_nonshared.a
echo "/* GNU ld script
   Use the shared library, but some functions are only in
   the static library, so try that secondarily.  */
OUTPUT_FORMAT(elf64-x86-64)
GROUP ( /$X86_LIBS/libc.so.6 $UTBOT_ALL/debs-install/usr/$X86_LIBS/libc_nonshared.a  AS_NEEDED ( /$X86_LIBS/ld-linux-x86-64.so.2 ) )" > $UTBOT_ALL/debs-install/usr/$X86_LIBS/libc.so

# Updating libpthread.so so that it contains valid path to libpthread_nonshared.a
echo "/* GNU ld script
   Use the shared library, but some functions are only in
   the static library, so try that secondarily.  */
OUTPUT_FORMAT(elf64-x86-64)
GROUP ( /$X86_LIBS/libpthread.so.0 $UTBOT_ALL/debs-install/usr/$X86_LIBS/libpthread_nonshared.a )" > $UTBOT_ALL/debs-install/usr/$X86_LIBS/libpthread.so
```

**Comment by rajgideon** [comment]
I am having the same issue **"mkdir: error while loading shared libraries: /lib/x86_64-linux-gnu/libpthread.so.0: invalid ELF header"** . I am running a Ubuntu 22.04 docker image in container.  after executing the script ./unpack_and_run_utbot.sh.  The Ubuntu container crashes and never stays up.  It appears the libraries are getting overwritten or destroyed. 
Please let me know if a fix is being planned for this issue.

**Comment by lancasterJie** [comment]
same issue in my ubuntu server, and re-install the OS, so frustrating~~~~~

**Comment by zhangzhili1112** [comment]
> same issue in my ubuntu server, and re-install the OS, so frustrating~~~~~

Hello, has this problem been solved?
